### ISSUE TITLE
Blood suckers kinda suck

### ISSUE THREAD

**Issue by Saratoubi** [yogstation13/Yogstation#14223]
### Severity

Inconvenience

### Short Description

Got my 2nd ever bloodsucker round, was going pretty well until I randomly went into a frenzy, then turned to ash...

### Reproduction

So to be perfectly honest I'm a bad blood sucker, my first round was embarassing to say the least, but this was low pop hours so I wanted to give it a try. I spawned as the virologist, stole a humanized monkey from genetics, then set up in my lab for a slow round of learning. I even had time to make my brain dead monkey friend a virus to regen blood. I soon hungered for more, and having learned that thralls are a thing, I wanted to go kidnap myself one. I kidnapped the a fellow medical employee, strapped her to the persuasion device, and then contacted a mentor to figure out what the fuck I was doing. After a little confusion I managed to turn her into my thrall upon which I immediately went into a frenzy. It was exactly at that point that my internet cut out, so after wait 1-2 minutes it came back. So then hoping to get out of said frenzy I sucked my little comatose friend, killed him instantly, then burst into ash perma died.
Btw the text to tell that your humanity had changed is so inconspicious. Playing blood suckers for the first time is like having to drive a fucking car while reading an instruction manual on how to drive said fucking car. You are just assaulted by walls of text, and the wiki page is very very unhelpful as it of now.
Also it was night time when I turned to ash. Thought blood suckers only hit true death when they got killed by the sun.

TL;DR
What I think happened:
1. Turned someone into my thrall, then turned them into my favorite thrall. Which I guess raised the humanity frenzy threshold to 280 blood.
2. Got frenzied by the sudden spike in threshold, and loss of blood (even though im pretty sure I had over 400 when I went frenzy)
3. Came back from internet outage, saw I was still frenzied, drank blood from nearby comatose monkey-human, trying to remedy that, who died instantly, thus raising my humanity threshold even more, and killing me instantly. (which by the way if thats how it works that fucking bullshit. You are frenzied so you drink blood instantly, thus killing whoever you are drinking, thus raising the threshold of your frenzy even more, rinse repeat until you turn to ash????)

### Round ID

41112

### BYOND Username

Saratoubi

**Comment by tattax** [comment]
You can suck alot of blood from a non catatonic person to replenish yourself quickly (especially on frenzy, which makes humanity not matter much till your next one), but sucking a cata monkey like you did has a /5 multiplier which makes you suck little blood. Also frenzy does burn damage overtime, which eventually final deaths you due to burn damage. I also had added the frenzy threshold to the status bar to make sure the humanity text wouldn't be overlooked

**Comment by tattax** [comment]
lots of time has passed and bloodsucker additions have come, closing this
